api/api-rules/violation_exceptions.list

```diff
@@ -530,6 +530,7 @@ API rule violation: names_match,k8s.io/kube-controller-manager/config/v1alpha1,K
 API rule violation: names_match,k8s.io/kube-controller-manager/config/v1alpha1,KubeControllerManagerConfiguration,ServiceController
 API rule violation: names_match,k8s.io/kube-controller-manager/config/v1alpha1,KubeControllerManagerConfiguration,StatefulSetController
 API rule violation: names_match,k8s.io/kube-controller-manager/config/v1alpha1,KubeControllerManagerConfiguration,TTLAfterFinishedController
+API rule violation: names_match,k8s.io/kube-controller-manager/config/v1alpha1,KubeControllerManagerConfiguration,ValidatingAdmissionPolicyController
```

Comment:
I remember there might be an agreement of not having new violation added but not quite sure.

Reply:
It is great not to add more violation exceptions. However, this is an unfortunate compromise to keep the KCM configuration naming convention. I think we will leave this decision to the API reviewer.